shared/src/main/scala/effekt/Namer.scala

```diff
@@ -456,9 +456,9 @@ trait NamerOps { self: Context =>
     scope.define(id.name, s)
   }
 
-  private[namer] def bind(s: TermSymbol): Unit = scope.define(s.name.name, s)
+  private[namer] def bind(s: TermSymbol): Unit = scope.define(s.name.localName, s)
 
-  private[namer] def bind(s: TypeSymbol): Unit = scope.define(s.name.name, s)
+  private[namer] def bind(s: TypeSymbol): Unit = scope.define(s.name.localName, s)
```

Comment:
I guess, in the long run you want to refactor scope to index by `Name` and not string, right?

Reply:
Yes this might be one of the use cases.